docs/UserGuide.md

```diff
@@ -227,15 +226,16 @@ Scroll down to the bottom and click on `recruitIn.jar`.
 </div>
 
 ### Viewing help : `help`
+[Table of contents](#table-of-contents)
 
 Shows a message explaining how to access the help page.
 
 ![help message](images/helpMessage.png)
 
 Format: `help`
 
```

Comment:
Perhaps you can include that the user can access the help menu by clicking on the Help button as well?